source/grid/reference_cell.cc

```diff
@@ -101,6 +104,42 @@ namespace ReferenceCell
       }
   }
 
+  template <int dim>
+  const ScalarPolynomialsBase<dim> &
+  get_linear_poly_space(const Type &reference_cell)
+  {
+    AssertDimension(dim, get_dimension(reference_cell));
+
+    static const TensorProductPolynomials<dim> poly(
+      Polynomials::generate_complete_Lagrange_basis(
+        QGaussLobatto<1>(2).get_points()));
```

Comment:
You can have it slightly shorter and with fewer indirections in the implementation by
```suggestion
      Polynomials::LagrangeEquidistant::generate_complete_basis(1));
```